### DISCUSSION TITLE
Need Windows binary from latest commit

### DISCUSSION THREAD

**Discussion by sabaatworld** [awawa-dev/HyperHDR#238] [General]
I want to try out the version with the most recent fixes but I'm unable to compile from source. Can anyone please provide a pre-built binary here for Windows? The issue that I'm encountering during build is that windows command prompt cannot find "make".

**Comment by awawa-dev** [comment]
Did you install VS2022 and all other components (including some of them in PATH during the installation) as it is stated in the wiki?
Binary setups are available in the **latest** build in github action:
![obraz](https://user-images.githubusercontent.com/69086569/156881693-12e0d58a-9fdf-4174-b904-3885925f5250.png)
  
  
They are at the bottom of the page. For Example:
![obraz](https://user-images.githubusercontent.com/69086569/156881750-0dfa5e71-36a0-43a9-b739-e5ad498933a2.png)

**Reply by sabaatworld** [reply]
I did install the dependencies but maybe I missed something among them. Does anything needs to be different on windows 11?

Thanks for pointing out that the artifacts are at the bottom. For some reason I never scrolled down.

**Reply by awawa-dev** [reply]
After first execution of cmake to configure the project in build folder, new *.sln (solution) file is created and can be open directly in VS2022. You can compile the project there.
